benchmark/opperf/nd_operations/nn_conv_operators.py

```diff
@@ -60,131 +81,286 @@ def run_pooling_operators_benchmarks(ctx=mx.cpu(), dtype='float32', profiler='na
-                                                                      "pad": (0, 0)}
-                                                                     ],
-                                                             warmup=warmup,
-                                                             runs=runs)
+            if large_tensor == 'on':
+                for pool1d_data in [(1, 1, 2**32), (2**31, 1, 3)]:
+                    pool1d_benchmark_res += run_performance_test([getattr(MX_OP_MODULE, "Pooling")],
+                                                                 run_backward=True,
+                                                                 dtype=dtype,
+                                                                 ctx=ctx,
+                                                                 profiler=profiler,
+                                                                 inputs=[{"data": pool1d_data,
+                                                                          "kernel": 3,
+                                                                          "pool_type": pool_type,
+                                                                          "global_pool": global_pool,
+                                                                          "stride": 1,
+                                                                          "pad": 1}
+                                                                        ],
+                                                                 warmup=warmup,
+                                                                 runs=runs)
+                for pool2d_data in [(2**29, 1, 3, 3), (2**28, 1, 4, 4)]:
+                    pool2d_benchmark_res += run_performance_test([getattr(MX_OP_MODULE, "Pooling")],
+                                                                 run_backward=True,
+                                                                 dtype=dtype,
+                                                                 ctx=ctx,
+                                                                 profiler=profiler,
+                                                                 inputs=[{"data": pool2d_data,
+                                                                          "kernel": (3, 3),
+                                                                          "pool_type": pool_type,
+                                                                          "global_pool": global_pool,
+                                                                          "stride": (1, 1),
+                                                                          "pad": (0, 0)}
+                                                                        ],
+                                                                 warmup=warmup,
+                                                                 runs=runs)
+            else:
```

Comment:
It seems the only difference between the if and else branch is the `inputs` argument. Can we only generate different inputs in the if/else branch and pass them to the same operator function?